spring-integration-sftp/src/test/java/org/springframework/integration/sftp/dsl/SftpTests.java

```diff
@@ -117,12 +119,14 @@ public void testSftpInboundStreamFlow() throws Exception {
 		assertNotNull(message);
 		assertThat(message.getPayload(), instanceOf(InputStream.class));
 		assertThat(message.getHeaders().get(FileHeaders.REMOTE_FILE), isOneOf(" sftpSource1.txt", "sftpSource2.txt"));
+		StreamUtils.copy((InputStream) message.getPayload(), new ByteArrayOutputStream());
```

Comment:
Well, my solution is actually like this:
```
((InputStream) message.getPayload()).close();
```
Because the problem with the file on Windows that we have an open stream to it. And, yeah..., exactly payload is `InputStream` in case of streaming inbound channel adapter.